tests/algorithms/test_blurpool.py

```diff
@@ -1,96 +1,159 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+"""Test the blurpool algorithm.
+
+Primitives are tested in test_blurpool.py
+"""
 import itertools
+from typing import List
+from unittest.mock import Mock
 
 import pytest
 import torch
 
-from composer.algorithms import blurpool
-from composer.algorithms.blurpool.blurpool import BlurPoolHparams
-from composer.trainer import TrainerHparams
-from tests.utils.trainer_fit import train_model
+from composer.algorithms import BlurPool, BlurPoolHparams
+from composer.algorithms.blurpool import apply_blurpool
+from composer.algorithms.blurpool.blurpool_layers import BlurConv2d, BlurMaxPool2d
+from composer.core import Event, State, surgery
+from composer.core.types import Logger, Tensors
+from composer.models import ComposerClassifier
+
+
+class ConvModel(torch.nn.Module):
```

Comment:
This model was specifically designed to test the blurpool algorithms, so scoping it to just this test, instead of making it globally available.